src/pages/stacks-blockchain/integrate-stacking.md

```diff
@@ -0,0 +1,366 @@
+
+- Generate Stacks accounts
+- Display stacking info
+- Verify stacking eligibility
+- Add stacking action
+- Display stacking status
+
+-> Alternatively to integration using JS libraries, you can [use the CLI](https://gist.github.com/kantai/c261ca04114231f0f6a7ce34f0d2499b).
+
+## Requirements
+
+First, you will need to understand the [Stacking mechanism](/stacks-blockchain/stacking).
+
+You will also need [NodeJS](https://nodejs.org/en/download/) `8.12.0` or higher to complete this tutorial. You can verify your installation by opening up your terminal and run the following command:
+
+```bash
+node --version
+```
+
+## Overview
+
+In this tutorial, we will implement the Stacking flow laid out in the [Stacking guide](/stacks-blockchain/stacking#stacking-flow).
+
+-> Check out the sample source code for this tutorial in this GitHub repository: [stacking-integration-sample](https://github.com/agraebe/stacking-integration-sample)
+
+## Step 1: Integrate libraries
+
+First install the stacks transactions library and an API client for the [Stacks 2.0 Blockchain API](/references/stacks-blockchain):
+
+```shell
+npm install --save @blockstack/stacks-transactions @stacks/blockchain-api-client c32check
+```
+
+-> The API client is generated from the [OpenAPI specification](https://github.com/blockstack/stacks-blockchain-api/blob/master/docs/openapi.yaml) ([openapi-generator](https://github.com/OpenAPITools/openapi-generator)). Many other languages and frameworks are be supported by the generator.
+
+## Step 2: Generating an account
+
+To get started, let's create a new, random Stacks 2.0 account:
+
+```js
+const {
+  makeRandomPrivKey,
+  privateKeyToString,
+  getAddressFromPrivateKey,
+  TransactionVersion,
+  StacksTestnet,
+  uintCV,
+  tupleCV,
+  makeContractCall,
+  bufferCV,
+  serializeCV,
+  deserializeCV,
+  cvToString,
+  connectWebSocketClient,
+  broadcastTransaction,
+  standardPrincipalCV,
+} = require('@blockstack/stacks-transactions');
+const {
+  InfoApi,
+  AccountsApi,
+  SmartContractsApi,
+  Configuration,
+  TransactionsApi,
+} = require('@stacks/blockchain-api-client');
+const c32 = require('c32check');
+
+const apiConfig = new Configuration({
+  fetchApi: fetch,
+  basePath: 'https://stacks-node-api.blockstack.org',
+});
+
+// generate rnadom key
+const privateKey = makeRandomPrivKey();
+
+// get Stacks address
+const stxAddress = getAddressFromPrivateKey(
+  privateKeyToString(privateKey),
+  TransactionVersion.Testnet
+);
+```
+
+-> Review the [accounts guide](/stacks-blockchain/accounts) for more details
+
+## Step 3: Display stacking info
+
+In order to inform users about the upcoming reward cycle, we need to obtain Stacking information:
+
+```js
+const info = new InfoApi(apiConfig);
+
+const poxInfo = await info.getPoxInfo();
+const coreInfo = await info.getCoreApiInfo();
+const blocktimeInfo = await info.getNetworkBlockTimes();
+
+console.log({ poxInfo, coreInfo, blocktimeInfo });
```

Comment:
Might be useful to include an example of the info response here.

Reply:
good point, added!